x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/schema/risk_score_apis.yml

```
@@ -227,6 +227,9 @@ components:
               type: boolean
             legacy_risk_engine_disabled:
               type: boolean
+            asset_criticality_installed:
```

This would be more consistent with the existing results: 
```suggestion
            asset_criticality_resources_installed:
```